django_ninja_crudl/endpoints/update.py

```
@@ -0,0 +1,102 @@
+"""CRUDL API base class."""
```

Ruff (0.8.2)

File `django_ninja_crudl/endpoints/update.py` is part of an implicit namespace package. Add an `__init__.py`.

(INP001)